src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -348,7 +363,8 @@
          to-token-id ""
          network-preferences (if token [] [(get-in collectible [:id :contract-id :chain-id])])
          gas-rates constants/gas-rate-medium
-         amount-in (send-utils/amount-in-hex amount (if token token-decimal 0))
+         to-hex (fn [v] (send-utils/amount-in-hex v (if token token-decimal 0)))
+         amount-in (to-hex amount)
```

Comment:
why did you need to do this?

Reply:
Because `to-hex` is used one more time below)